package.json

```diff
@@ -14,6 +14,7 @@
   },
   "homepage": "https://github.com/meetecho/janus-gateway#readme",
   "dependencies": {
-    "webrtc-adapter": "8.0.0"
+    "webrtc-adapter": "8.0.0",
+    "eslint": "5.0.1"
```

Comment:
please move `eslint` to `devDependencies` since it is only needed for developing and testing purposes

Reply:
Will do.